File: src/main/java/com/sandjelkovic/dispatchd/content/trakt/dto/ShowUpdateTrakt.java
package com.sandjelkovic.dispatchd.content.trakt.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.RequiredArgsConstructor;

import java.time.ZonedDateTime;

/**
 * @author ${sandjelkovic}
 * @date 5.2.17.
 */
@Data
@RequiredArgsConstructor
@AllArgsConstructor
public class ShowUpdateTrakt {
	@JsonProperty("updated_at")
	private ZonedDateTime updatedAt;
	private TvShowTrakt show;
}

File: content-service/src/main/kotlin/com/sandjelkovic/dispatchd/contentservice/trakt/dto/dto/ShowUpdateTrakt.kt
package com.sandjelkovic.dispatchd.contentservice.trakt.dto.dto

import com.fasterxml.jackson.annotation.JsonProperty
import java.time.ZonedDateTime

/**
 * @author sandjelkovic
 * @date 28.1.18.
 */
data class ShowUpdateTrakt(
        @JsonProperty("updated_at")
        val updatedAt: ZonedDateTime? = null,
        val show: ShowTrakt? = null
)
